config/compiler/BUILD.gn

```diff
@@ -1564,16 +1564,6 @@ config("default_warnings") {
         cflags += [ "-Wno-unqualified-std-cast-call" ]
       }
 
-      if (!is_nacl && !(is_chromeos ||
-                        default_toolchain == "//build/toolchain/cros:target") &&
-          clang_major_version >= 15) {
```

Comment:
this change `clang_major_version >= 15)` satisfies my problem where I'm using clang 14. removed local patch and no error from the build.